src/fides/api/util/tcf/tc_string.py

```diff
@@ -214,13 +219,19 @@ def convert_to_fides_preference(
         # Check if there's an opt_in encoded in the string.  Otherwise, we assume the user is opting out.
         preference: bool = tc_string_preferences.get(identifier, False)
 
-        if isinstance(preference_class, TCFVendorSave):
-            # Vendors are currently saved as strings in our db
-            identifier = str(identifier)
+        if preference_class == TCFVendorSave:
+            # Vendors are currently saved as strings in our db.
+            # We also need to add the gvl prefix here to turn this into
+            # a universal vendor id.
+            vendor_id: str = add_gvl_prefix(str(identifier))
```

Comment:
yup, this assumption is good, i.e. all TC string specified vendors can be assumed to be GVL vendors 👍